src/column.ts

```diff
@@ -120,21 +120,27 @@ export class ColumnFactory {
 
 export class RawColumn {
     public cellLength: number = 0;
-    constructor(public columnValues: string[]) { 
+    public cellLengths: number[] = [];

```

Comment:
There shouldn't be an additional array to do this job, I think that cellLength could be used.